run.go

```diff
@@ -97,18 +97,39 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	if err != nil {
 		return err
 	}
+	boundMounts := []specs.Mount{}
 	defer func() {
 		if err2 := b.Unmount(); err2 != nil {
 			logrus.Errorf("error unmounting container: %v", err2)
 		}
 	}()
 	g.SetRootPath(mountPoint)
 	g.SetProcessTerminal(true)
+	g.RemoveLinuxNamespace("network")
 	spec := g.Spec()
 	if spec.Process.Cwd == "" {
 		spec.Process.Cwd = DefaultWorkingDir
 	}
 	mounts := options.Mounts
+	for _, boundFile := range []string{"/etc/hosts", "/etc/resolv.conf"} {
+		override := false
+		for _, mount := range mounts {
+			if mount.Destination == boundFile {
+				// Already have an override for it, so skip this one.
+				continue
+			}
+		}
+		if override {
```

Comment:
Unless I've been staring at the screen too long today, I'm not seeing how override could be true here.  Do we need this if or even the boolean?